### ISSUE TITLE
jstack /path/to/java /path/to/coredump no longer valid in JDK 14

### ISSUE THREAD

**Issue by marcin-chwedczuk** [tldr-pages/tldr#4177]
Looks like in JDK 9 a new tool called `jhsdb` was introduced. To get a thread dump from a coredump file you need to call:
```
jhsdb jstack --core /cores/core.56128 --exe /usr/local/Cellar/openjdk/14.0.1/bin/java
```
instead of 
```
- Print stack traces from java core dump:
    jstack /usr/bin/java file.core
```
as TLDR suggests.

There is no TLDR page for `jhsdb` command. Maybe it's time to add it?

I tested this on Oracle and Open JDK 14 on macOS 10.15.

**Comment by owenvoke** [comment]
Hi @marcin-chwedczuk, thanks for mentioning this! Do you want to have a go adding the `jhsdb` page yourself?

Pages are best created by people who understand the command they are about.

If you do, take a quick read of [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md), which guides you through the process of creating a new page here. Then, just simply send a pull request to this repository - and if you need any assistance, just ask :-)

**Comment by bl-ue** [comment]
@marcin-chwedczuk do want to fix this?

**Comment by gutjuri** [comment]
Can this issue be closed now that #6643 has been merged?

**Comment by owenvoke** [comment]
@gutjuri, I think this is safe to close. 👍🏻
